### DISCUSSION TITLE
Is it possible to make it work on network, for Non WSL vms

### DISCUSSION THREAD

**Discussion by TSEVER** [dorssel/usbipd-win#830] [Ideas]
Hello I was so excited when I see the project, thank you. It is a brilliant idea. I thought if it will work with WSL why not with the Hyper-V vm and failed :(. Is there any way to make it work From Hyper-V host to Hyper-V Windows VM, or Windows/ Linux Desktop to Hyper-V Windows VM in the samenetwork?

**Comment by dorssel** [comment]
This project is server-side only. Yes, it works with any networked client (including Hyper-V). But this project does not have any client side drivers (see README). Linux comes with USB/IP support out-of-the-box. For other OSs, you will need to find client drivers elsewhere.
